src/fu-main.c

```diff
@@ -1493,7 +1493,7 @@ fu_main_on_name_lost_cb (GDBusConnection *connection,
 			 gpointer user_data)
 {
 	FuMainPrivate *priv = (FuMainPrivate *) user_data;
-	g_debug ("FuMain: lost name: %s", name);
+	g_warning ("another service has claimed the dbus name %s", name);
```

Comment:
isn't this is a legitimate code path when we replace the system version of fwupd with one compiled locally?

Reply:
Can one compiled locally actually take over?  Locally compiled one needs messing with default place to find dbus services (IE look at compilation page what we have down https://github.com/fwupd/fwupd/wiki/Compilation)

Reply:
And furthermore - if that was the case, a system one was running and a locally compiled one takes over, I would certainly rather see that in logs that don't have verbose turned on..

Reply:
True.